uvtools/dspec.py

```diff
@@ -1998,7 +1998,8 @@ def fit_solution_matrix(weights, design_matrix, cache=None, hash_decimal=10, fit
 
 
 def dpss_operator(x, filter_centers, filter_half_widths, cache=None, eigenval_cutoff=None,
-        edge_suppression=None, nterms=None, avg_suppression=None, xc=None, hash_decimal=10):
+        edge_suppression=None, nterms=None, avg_suppression=None, xc=None, hash_decimal=10,
+        xtol=0.1):
```

Comment:
this feels like a very high xtol for default. I'd recommend making it closer to machine precision (1e-6 or less) and forcing the user to acknowledge to realize there's something weird and deal with it.

Reply:
I've changed this to `1e-3` which is what we see out of the lstbinner

https://github.com/HERA-Team/hera_cal/issues/698